e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -54,4 +57,26 @@ describe('Customization', () => {
             expect(config.TeamSettings.SiteName).to.eq(siteName);
         });
     });
+
+    it('SC20339 - Can change Enable Custom Branding setting', () => {
```

Comment:
Test failed when the initial value is true; clicking "true" will not enable "Save" button.  One way is to preset "EnableCustomBrand" to false by doing `cy.apiUpdateConfig` in this particular test.

Reply:
The latest change will still fail when the initial value is true.  I suggest to have here:
```javascript
cy.apiUpdateConfigBasic({TeamSettings: {EnableCustomBrand: false}});
cy.reload();

cy.findByTestId('TeamSettings.EnableCustomBrand').should('be.visible').within(() => {...
```